packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -256,8 +256,8 @@ const Dropdown = ({
 
     React.useEffect(() => {
         if (!initial_render.current && !is_list_visible && value) dropdown_ref.current.focus();
-    }, [is_list_visible, value]);
-
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [is_list_visible]);
```

Comment:
this change will affect all dropdowns across the app (not only in trading assessment). is it intended? @likhith-deriv @aizad-deriv

Reply:
Yes it was intentional because it keeps on focusing after each render. The changes were made to solve the issue.